test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -593,6 +593,26 @@ def func(tensor):
         tensor = common_utils.get_whitenoise(sample_rate=44100)
         self._assert_consistency(func, tensor)
 
+    def test_character_edit_distance(self):
+
+        def func(tensor):
+            ref = "abc"
+            hyp = "bcd"
+            return F.character_edit_distance(ref, hyp)
+
+        dummy = torch.zeros(1, 1)
+        self._assert_consistency(func, dummy)
+
+    def test_word_edit_distance(self):
+
+        def func(tensor):
+            ref = ["hello", "world"]
+            hyp = ["hello", "planet"]
+            return F.word_edit_distance(ref, hyp)
+
+        dummy = torch.zeros(1, 1)
+        self._assert_consistency(func, dummy)
```

Comment:
Similar to the above, instead of `_assert_consistency`, implement the correct logic.

Reply:
I've fixed it in the new commit.